src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <section class="calendar calendar--start-day-sun calendar--month-length-28">
```

Comment:
I like the fact that you want to bring semantics to your html but in this particular case, section doesn't make much sense. I would prefer to see div here

[section tag use cases](https://www.w3schools.com/tags/tag_section.asp)